package.json

```diff
@@ -6,7 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lint:js" : "npm run lint"
```

Comment:
This works for now, but I'm thinking that once this and the CSS linting are set up, we'll want to use `npm-run-all` to run each linting script in sequence. Here's how I have it set up in my eleventy starter: https://github.com/dustin-jw/eleventy-starter/blob/main/package.json#L16-L26

Reply:
Before I commit it I want sure I I'm understanding this correctly. I should add `npm-run-all ` as a dev dependency and the the scripts should look something like this?  
``` 
    "lint": "run-s lint:js",
    "lint:js": "next lint"
```

Reply:
Yep, you got it!

Reply:
change committed